lib/model.js

```diff
@@ -3143,7 +3143,7 @@ class Model {
       }
       return values;
     }
-    return this.dataValues;
+    return Utils._.clone(this.dataValues);
```

Comment:
This is also what I did in the first place, though I thought it is a bit odd decision to expose internal data structure through the public API which says `Convert the instance to a JSON representation`.